modules/ossm-cr-example.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-cr-example_{context}"]
+= {ProductName} custom resource
+
+[NOTE]
+====
+The `istio-system` namespace is used an example throughout the {ProductShortName} documentation, but you can use other namespaces as necessary.
+====
+
+To deploy the {ProductShortName} control plane, you must create a custom resource. A _custom resource_ allows you to introduce your own API into a Kubernetes project or cluster. You create a custom resource yaml file that defines the project parameters and creates the object. This example custom resource yaml file contains all of the supported parameters and deploys {ProductName} {ProductVersion} images based on Red Hat Enterprise Linux (RHEL).
+
+[IMPORTANT]
+====
+The 3scale Istio Adapter is deployed and configured in the custom resource file. It also requires a working 3scale account (link:https://www.3scale.net/signup/[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/infrastructure/onpremises-installation[On-Premises]).
+====
+
+.Full example istio-installation.yaml
+
+[source,yaml]
+----
+  apiVersion: maistra.io/v1
+  kind: ServiceMeshControlPlane
+  metadata:
+    name: basic-install
+
+    threeScale:
+      enabled: false
+
+    istio:
+      global:
+        proxy:
+          resources:
+            requests:
+              cpu: 100m
+              memory: 128Mi
+            limits:
+              cpu: 500m
+              memory: 128Mi
+        multitenant: true
+
+      gateways:
+        istio-egressgateway:
+          autoscaleEnabled: false
+        istio-ingressgateway:
+          autoscaleEnabled: false
+          ior_enabled: false
+
+      mixer:
+        policy:
+          autoscaleEnabled: false
+
+        telemetry:
+          autoscaleEnabled: false
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1G
+            limits:
+              cpu: 500m
+              memory: 4G
+
+      pilot:
+        autoscaleEnabled: false
+        traceSampling: 100.0
+
+      kiali:
+       dashboard:
+          user: admin
+          passphrase: admin
+      tracing:
+        enabled: true
```

Comment:
This needs Jaeger info:
    # to disable tracing (i.e. jaeger)
    tracing:
      enabled: false
      jaeger:
        # simple, all-in-one strategy
        template: all-in-one
        # production strategy, utilizing elasticsearch
        #template: production-elasticsearch
        # if required. only one instance may use agentStrategy=DaemonSet
        #agentStrategy: DaemonSet

Reply:
@brian-avery Should this be with the rest of the tracing parameters in [modules/ossm-cr-tracing.adoc](https://github.com/openshift/openshift-docs/pull/15681/files#diff-6744fe88ebb5e216d95b9a28bad217d5)?

Reply:
Discussed this with @brian-avery and we updated the tracing cr information with a Tracing/Jaeger example, renamed the adoc/headers appropriately, and updated the assembly appropriately. Changes coming in the next PR update.